config.ts

```diff
@@ -71,7 +71,7 @@ config.app_config.APP_SIDEMENU_DEFAULTS.title = "Debug App";
 config.app_config.APP_SKINS.defaultSkinName = SKINS.defaultSkin.name
 config.app_config.APP_SKINS.available = [SKINS.defaultSkin, SKINS.debug]
 
-config.app_config.APP_THEMES.available = ["default", "professional", "pfr"]
+config.app_config.APP_THEMES.available = ["default", "professional", "pfr", "plh_kids_kw"]
```

Comment:
```suggestion
config.app_config.APP_THEMES.available = ["default", "professional", "pfr", "plh_facilitator_mx", "early_family_math", "plh_kids_kw"]
```

Making other themes from `src/theme/themes/_index.scss` available as well now that we're at it.